README.md

```diff
@@ -2,14 +2,34 @@
 
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
+  
+a. On the 'Average Case Running Time' graph in the sorting lecture slides, you can compare 'Timsort' and 'Heapsort'. For n under ~80,000, Timsort is faster than Heapsort. Then Heapsort jumps above Timsort and as they continue to grow, they even out and are very similar as n approached infinity. While they grow the same as they are approaching large sizes of n, for a more realistic size of n, one algorithm would be faster than the other.
```

Comment:
But the asymptotic complexity isn't about what is faster, it's only about the growth.